apps/frontend/src/pages/projects/styles.module.css

```diff
@@ -1,5 +1,8 @@
 .label {
```

Comment:
class names should also be consistent
```suggestion
.title {
```